pkg/models/testconfigoptions.go

```diff
@@ -0,0 +1,23 @@
+package models
+
+import "time"
+
+type TestConfigOptions struct {
+	Path               *string
+	ProxyPort          *uint32
+	AppCmd             *string
+	Tests              *map[string][]string
+	AppContainer       *string
+	NetworkName        *string
+	Delay              *uint64
+	BuildDelay         *time.Duration
+	PassThroughPorts   *[]uint
+	ApiTimeout         *uint64
+	GlobalNoise        *GlobalNoise
+	TestSetNoise       *TestsetNoise
+	CoverageReportPath *string
+	WithCoverage       *bool
+	ConfigPath         *string
+	IgnoreOrdering     *bool
+	PassThroughHosts   *[]Filters
```

Comment:
instead of redefining it again better to make a struct over the pre-existing struct , this way same struct can be used in gettestconfig as well as `test`

Reply:
The two structs TestOptions and TestConfigOptions have some variances in properties, which will be needed to be set to NULL if we combine make a struct over the pre existing one if i am not wrong